source/opt/set_spec_constant_default_value_pass.cpp

```diff
@@ -248,5 +250,85 @@ bool SetSpecConstantDefaultValuePass::Process(ir::Module* module) {
   return modified;
 }
 
+// A helper function for parsing spec constant default value string. Returns
+// true if the char pointed by the given pointer is considered as the end of
```

Comment:
The blurb should not say what context the function is used in, since that tends to go stale. Just say what it computes.